src/Apps/Artwork/Components/ArtistInfo.tsx

```diff
@@ -114,24 +142,39 @@ export class ArtistInfo extends Component<ArtistInfoProps> {
                   />
                 </>
               )}
+              {showArtistInsightsButton && (
+                <Button
+                  onClick={this.toggleArtistInsights}
+                  width={130}
```

Comment:
I think instead of specifying a static width here, we should be making sure the button hugs its inner text (as opposed to stretching to fit it's parent container).

I'd probably do that be wrapping the button in a 

    <Flex flexDirection="column" alignItems="flex-start">

Reply:
Nicole said animation wasn't necessary